post_installation_configuration/storage-configuration.adoc

```diff
@@ -24,19 +24,19 @@ cluster to your requirements, including storage configuration.
 [id="post-install-dynamic-provisioning"]
 == Dynamic provisioning
 
-include::modules/dynamic-provisioning-about.adoc[leveloffset=+3]
+include::modules/dynamic-provisioning-about.adoc[leveloffset=+2]
```

Comment:
@ahardin-rh Can you check this file? It had some incorrect levels and I just want a sanity check that they indentation/levels are still what you were expecting. Thanks!

Preview: https://fixin-build-issues--ocpdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/storage-configuration.html

Reply:
Your changes LGTM! Thanks!